open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/resources/application.properties

```diff
@@ -39,18 +53,9 @@ ldap.url=ldap://localhost:389
 authentication.source=db
 authentication.mode=token
 
-#gaiandb
-gaiandb.protocol=jdbc:derby://
-gaiandb.host=localhost
-gaiandb.port=6414
-gaiandb.db=gaiandb
-gaiandb.proxyUser=gaiandb
-gaiandb.proxyPwd=passw0rd
-gaiandb.schema=GAIANDB
-
-# omas server connection details
-omas.server.name=omas
-omas.server.url=http://localhost:8080
-open.lineage.server.url=http://localhost:8082
-open.lineage.server.name=open-lineage
-open.lineage.graph.source=MAIN
\ No newline at end of file
+# Previously, omas server connection details were specified as properties. These values are now defined in the ui server config.
```

Comment:
we will need to remove these from the docker-compose & k8s charts. Will look for docs in review to define new things that need adding. Currently these parms configure the connectivity used

Reply:
Tracking in https://github.com/odpi/egeria/issues/2266 - we will persue in an independent issue